apollo-ios-codegen/Sources/ApolloCodegenLib/Constants.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public enum Constants {
+  public static let CodegenVersion: String = "1.15.3"
```

Comment:
I'm not clear on why we need this new constants file. The version of Apollo-ios and codegen should always be the same. 

Do we just need to add this because they are going to be split into separate repos and the codegen won't have access to the constant from the Apollo library?

But wasn't the codegen already pointing to the version of Apollo though? In order to do the version mismatch validation? Why is this different than that?

Reply:
The existing constants files arent available to the `ApolloCodegenLib` so this seemed like the appropriate way to provide the version number for that library to use with the new `default` case for this work. Currently the generated `Package.swift` is not using exact version its a `from: 1.0.0`

Reply:
Gotcha. What I was thinking of is only in the CLI, not in the `ApolloCodegenLib`.